src/plugins/discover/public/application/main/utils/get_fetch_observeable.test.ts

```diff
@@ -90,7 +89,7 @@ describe('getFetchObservable', () => {
       fetchfnMock();
     });
     autoRefreshFetch$.next(jest.fn());
-    jest.runAllTimers();
+    await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
Quick question, why this change? didn't it work anymore?

Reply:
It is due to the new implementation of the RxJS Scheduler: it enters an infinite loop of timers. However, I recently discovered the `fakeSchedulers` helper. I changed the test to use the `advance` API instead of an actual delay.

Just FYI: in the same file there's a test that is not using fake timers and that's the only reason it takes 100ms to run (due to the internal debounce, in case you want to fix it in a follow up PR).

Reply:
thx @afharo !